File: hackerrank/C++/STL/Vector-Sort/solve.cpp
/**
 *      Author: Kessyus Fofano
 *      Email: kessyus@gmail.com
**/

#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */

    int size {};
    cin >> size;
    vector<int> v;

    for(int i=0; i<size; ++i){
        int temp;
        cin >> temp;
        v.push_back(temp);
    }

    sort(v.begin(),v.end());

    for(int i=0; i<size; ++i){
        cout << v[i] << " ";
    }

    return 0;
}

File: hackerrank/Python/Symmetric Difference/solve.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
n = int(input())
lista1 = list(map(int, input().split()))
m = int(input())
lista2 = list(map(int, input().split()))

lista1 = set(lista1)
lista2 = set(lista2)

lista3 = sorted(lista1.symmetric_difference(lista2))

for i in list(lista3):
    print(i, sep="\n")
